Atomic/__init__.py

```diff
@@ -1,11 +1,11 @@
 import sys
-import os
 from .pulp import PulpServer
 from .config import PulpConfig
 from .atomic import Atomic
```

Comment:
`PulpConfig` and `Atomic` are imported but unused, should these imports be removed?